### ISSUE TITLE
2-ethylhydracylic aciduria

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6367]
**Preferred term label:**
2-ethylhydracylic aciduria

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increased concentration of 2-ethylhydracylic acid in the urine.

Isolated excretion of 2-ethylhydracrylic acid (2-methylbutyrylglycine; 2-MBG) is the hallmark of short/branched-chain acyl-CoA dehydrogenase deficiency (SBCADD), a defect in the proximal pathway of L-isoleucine oxidation (PMID: 15615815 ). 

**Parent term (use hpo.jax.org/app)**
Elevated urinary carboxylic acid HP:0040156

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Short/branched-chain acyl-CoA dehydrogenase deficiency

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I have added the term for 2-ethylhydracylic aciduria

I have removed the part about " (2-methylbutyrylglycine; 2-MBG) " because I do not think it is a synonym
2-Ethylhydracrylic-acid -- C5H10O3
https://pubchem.ncbi.nlm.nih.gov/compound/2-Ethylhydracrylic-acid
2-Methylbutyrylglycine - C7H13NO3
https://pubchem.ncbi.nlm.nih.gov/compound/2-Methylbutyrylglycine

Please open a new issue if we still need to do something about 2-Methylbutyrylglycine
